### ISSUE TITLE
Keeping your units straight

### ISSUE THREAD

**Issue by Breinholm-coder** [homewizard/api-documentation#60]
You are using kWh for entities named with power - Power is measured in W or kW -
kWh is energy.
Example from your documentation:
"total_power_import_kwh | Number | The power usage meter reading for all tariffs in kWh"
It should be:
"total_energy_import_kWh | Number | The energy usage meter reading for all tariffs in kWh"

**Comment by DCSBL** [comment]
You are right, this is a thing from the past which is hard to adjust; We can't simply update the API and assume implementations to pick that up. But the description itself has to be correct, so this has been adjusted in #61

Thanks for pointing out!
